packages/helpers/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 name: .intro
 route: /helpers
-menu: 6.Helpers
+menu: Helpers
```

Comment:
actually that number is required for now, cuz the order for side bar menu items are not supported for now with docz

Reply:
ok, so i'm gonna add another parameter to command, to let the user decide for the menu name, til we find out better way.